!Command
positional: []
named: []
command:
- SanitizeMe_GUI.py
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmp8m0lewyk/bin/SanitizeMe_GUI.py\"\
  , line 81, in <module>\n    sys.exit(main())\n  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/gooey/python_bindings/gooey_decorator.py\"\
  , line 124, in inner2\n    return payload(*args, **kwargs)\n  File \"/tmp/tmp8m0lewyk/bin/SanitizeMe_GUI.py\"\
  , line 34, in main\n    args = cli.parse_args()\n  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/gooey/python_bindings/gooey_parser.py\"\
  , line 116, in parse_args\n    return self.parser.parse_args(args, namespace)\n\
  \  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/gooey/python_bindings/gooey_decorator.py\"\
  , line 91, in run_gooey\n    from gooey.gui import application\n  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/gooey/gui/application.py\"\
  , line 5, in <module>\n    import wx\n  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/wx/__init__.py\"\
  , line 17, in <module>\n    from wx.core import *\n  File \"/tmp/tmp8m0lewyk/lib/python3.8/site-packages/wx/core.py\"\
  , line 12, in <module>\n    from ._core import *\nImportError: libXxf86vm.so.1:\
  \ cannot open shared object file: No such file or directory\n"
generated_using:
- --help
